velox/exec/tests/OperatorUtilsTest.cpp

```diff
@@ -206,3 +207,56 @@ TEST_F(OperatorUtilsTest, gatherCopy) {
 TEST_F(OperatorUtilsTest, makeOperatorSpillPath) {
   EXPECT_EQ("spill/task_1_100", makeOperatorSpillPath("spill", "task", 1, 100));
 }
+
+TEST_F(OperatorUtilsTest, wrap) {
+  auto rowType = ROW({
+      {"bool_val", BOOLEAN()},
+      {"tiny_val", TINYINT()},
+      {"small_val", SMALLINT()},
+      {"int_val", INTEGER()},
+      {"long_val", BIGINT()},
+      {"ordinal", BIGINT()},
+      {"float_val", REAL()},
+      {"double_val", DOUBLE()},
+      {"string_val", VARCHAR()},
+      {"array_val", ARRAY(VARCHAR())},
+      {"struct_val", ROW({{"s_int", INTEGER()}, {"s_array", ARRAY(REAL())}})},
+      {"map_val",
+       MAP(VARCHAR(),
+           MAP(BIGINT(),
+               ROW({{"s2_int", INTEGER()}, {"s2_string", VARCHAR()}})))},
+  });
+
+  for (int32_t iter = 0; iter < 20; ++iter) {
+    VectorFuzzer fuzzer({}, pool());
+    auto vector = CHECK_NOTNULL(fuzzer.fuzzFlat(rowType));
+    auto rowVector = vector->as<RowVector>();
+
+    const int32_t wrapVectorSize =
+        iter == 0 ? 0 : 1 + folly::Random().rand32(rowVector->size());
+    BufferPtr wrapIndicesBuffer = allocateIndices(wrapVectorSize, pool_.get());
```

Comment:
Use makeIndices method from the base class.

```
  BufferPtr makeIndices(
      vector_size_t size,
      std::function<vector_size_t(vector_size_t)> indexAt) const;
```

Reply:
Thanks!